mini

```diff
@@ -27,6 +27,7 @@ case "$COMMAND" in
     test)
         cargo test --manifest-path=tooling/minitest/Cargo.toml $CARGOFLAGS "$@"
         cargo test --manifest-path=tooling/minimize/Cargo.toml $CARGOFLAGS "$@"
+        cargo test --manifest-path=tooling/miniutil/Cargo.toml $CARGOFLAGS "$@"
```

Comment:
Please move this up so it is tested first -- this is the most fundamental of the crates, after all.

Reply:
done